libs/ledger-live-common/src/families/aptos/getTransactionStatus.test.ts

```diff
@@ -0,0 +1,283 @@
+import BigNumber from "bignumber.js";
+import { createFixtureAccount } from "../../mock/fixtures/cryptoCurrencies";
+import createTransaction from "./createTransaction";
+import getTransactionStatus from "./getTransactionStatus";
+import {
+  AmountRequired,
+  FeeNotLoaded,
+  GasLessThanEstimate,
+  InvalidAddress,
+  InvalidAddressBecauseDestinationIsAlsoSource,
+  NotEnoughBalance,
+  RecipientRequired,
+} from "@ledgerhq/errors";
+import { SequenseNumberTooNewError, SequenseNumberTooOldError, TransactionExpiredError } from "./errors";
+
+describe("getTransactionStatus Test", () => {
+  it("should return errors for AmountRequired", async () => {
+    const account = createFixtureAccount();
+    const transaction = createTransaction();
+
+    transaction.fees = new BigNumber(2);
+    transaction.recipient = "0x" + "0".repeat(64);
+
+    const result = await getTransactionStatus(account, transaction);
+
+    const expected = {
+      errors: {
+        amount: new AmountRequired(),
+      },
+      warnings: {},
+      estimatedFees: new BigNumber(2),
+      amount: new BigNumber(0),
+      totalSpent: new BigNumber(2),
+    };
+
+    expect(result).toEqual(expected);
+  });
+
+  it("should return errors for FeeNotLoaded", async () => {
+    const account = createFixtureAccount();
+    const transaction = createTransaction();
```

Comment:
Since transaction is created with an external method, I think we need to make sure that fee is null / undefined by manually setting it before the expect clause

I'd even go as far as not using createFixtureAccount and createTransaction method but that can be discussed